src/yadism/coefficient_functions/light/nlo/g1.py

```diff
@@ -0,0 +1,27 @@
+import numba as nb
+import numpy as np
+from eko.constants import TR, CF
+
+from . import f2
+
+
+@nb.njit("f8(f8,f8[:])", cache=True)
+def ns_reg(z, _args):
```

Comment:
@Radonirinaunimi here we should import this piece from `f3.ns_reg` eq 19 and paragraph below of [here](https://arxiv.org/pdf/2210.12014.pdf)

Reply:
I could not find from the cited references the explicit equation for such an equivalence relation, but upon doing such a change the results are exactly the same as before so this is good enough for me.

Since these equivalence relations are valid up to NNLO it would be even much easier to implement the NNLO $g_1$ contributions as we will not have to implement the lengthy equations in the old paper.